### PR TITLE
Fix/remove dashboard api

### PR DESCRIPTION
**Pull request by priyankatechneai**
Removed Dashboard api

### CODE REVIEW

File: src/screens/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -129,7 +82,8 @@ export default function HrDashboard(props) {
   async function get() {
     const id = sessionStorage.getItem("id");
     await getData(id).then((res) => {
-      if (res.status == 200) {
+      console.log("res", res);
```
Comment (AmitSolnke) [inline src/screens/Dashboard/Dashboard.js:85]: console not removed

File: src/screens/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -332,63 +241,50 @@ export default function HrDashboard(props) {
     });
   };
 
+  const loadData = () => {
```
Comment (AmitSolnke) [inline src/screens/Dashboard/Dashboard.js:244]: Use try catch on all api calling.
check for res.data.status as well

File: src/App.css
Comment (AmitSolnke) [inline src/App.css:1]: Kindly work on issue discussed verbally.

File: src/screens/Dashboard/Dashboard.js (hunk 1 above)
Comment (priyankatechneai) [reply src/screens/Dashboard/Dashboard.js:85]: resolved

File: src/screens/Dashboard/Dashboard.js (hunk 2 above)
Comment (priyankatechneai) [reply src/screens/Dashboard/Dashboard.js:244]: it will crash and give promise error while using try and catch

File: src/App.css
Comment (priyankatechneai) [reply src/App.css:1]: resolved

**Pull request closed without merging**